modules/composer-ffmpeg/src/test/java/org/opencastproject/composer/impl/ComposerServiceTest.java

```diff
@@ -353,14 +353,49 @@ public void testComposite() throws Exception {
             .get(1));
 
     Job composite = composerService.composite(outputDimension, Option.option(lowerLaidOutElement), upperLaidOutElement,
-            watermarkOption, "composite.work", "black");
+            watermarkOption, "composite.work", "black", null); // null or "both" means that both tracks are checked for audio and both audio tracks are mixed into the final composite if they exist
```

Comment:
Suggestion: Just replace _null_ by _"both"_ and remove the comment. I would recommend to not allow _null_ as actual parameter in composerService.composite() as you can expect clients to correctly chose between "lower", "upper" and "both".